static/js/components/Learner.js

```diff
@@ -62,12 +63,21 @@ export default class Learner extends React.Component {
 
     if (! R.isEmpty(dashboard)) {
       return dashboard.programs.map(program => (
-        <StaffLearnerInfoCard
-          program={program}
-          key={program.title}
-          setShowGradeDetailDialog={setShowGradeDetailDialog}
-          dialogVisibility={ui.dialogVisibility}
-        />
+        <div key={program.title}>
```

Comment:
Can we use `program.id` here instead? `title` isn't technically guaranteed to be unique